sdk/support/Microsoft.Azure.Management.Support/src/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This should be unnecessary, given the same file in the parent directory

Reply:
Removed.